### PR TITLE
Adds randomly generated mime names

### PR DESCRIPTION
**Pull request by CheeseWizard2**
## About The Pull Request

Adds 26 new names for mimes that can be found by randomizing your name in the character editor.

## Why It's Good For The Game

There's currently 5 times as many random clown names as there are random mime names (124 to 24, specifically). This PR helps close the gap, which adds variety to the names of mime players, whether they're hardcore random or their player just couldn't think of a name.

## Changelog

:cl:
add: 26 mime names to mime.txt

/:cl:

### CODE REVIEW

File: strings/names/mime.txt [hunk 1]
```diff
@@ -1,24 +1,50 @@
+Ceci N'est Pas Une Mime
+Cousteau
+Eastern Standard Mime
```
Comment (Time-Green) [inline strings/names/mime.txt:3]: Godtier

File: strings/names/mime.txt [hunk 2]
```diff
@@ -1,24 +1,50 @@
...
 Mr Mute
 Mute
+Nickel and Mime
+The Noiseless Wonder
 Omerta
+One Hand Clapping
+Oreo
 Oui Oui
 Pantomime
+Parle Francais
```
Comment (SuperNovaa41) [inline strings/names/mime.txt:27]: if you want to be grammatically correct here its `Parles` or `Parlez`

Comment (CheeseWizard2) [reply strings/names/mime.txt]: Added, thanks!

File: strings/names/mime.txt [hunk 3]
```diff
@@ -1,24 +1,50 @@
+Ceci N'est Pas Une Mime
```
Comment (dragomagol) [inline strings/names/mime.txt:1]: lmao

**Pull request merged**